blocks/single-column-layout-block/layouts/single-column-regular/default.jsx

```diff
@@ -5,25 +5,29 @@ import { Stack } from "@wpmedia/arc-themes-components";
 const LAYOUT_CLASS_NAME = "b-single-column-regular";
 
 const SingleColumnRegular = ({ children }) => {
-	const [navigation, main, footer] = React.Children.toArray(children);
+	const [navigation, body, fullWidth, footer] = React.Children.toArray(children);
 
-	if (!navigation && !main && !footer) {
+	if (!navigation && !body && !fullWidth && !footer) {
 		return null;
 	}
+
 	return (
 		<Stack className={LAYOUT_CLASS_NAME}>
 			{navigation ? (
-				<Stack role="banner" className={`${LAYOUT_CLASS_NAME}__navigation`}>
+				<Stack as="header" role="banner" className={`${LAYOUT_CLASS_NAME}__navigation`}>
```

Comment:
No need for the `role` attribute now we are using semantic elements

```suggestion
				<Stack as="header" className={`${LAYOUT_CLASS_NAME}__navigation`}>
```

Reply:
@matthewroach  Great catch! Thanks!